README.rst

```diff
@@ -22,16 +22,16 @@ However, an equivalent widely-used package does not exist for X-ray timing: to d
 Stingray aims not only to fill that gap, but also to provide implementations of the most advanced spectral timing techniques available in the literature.
 The ultimate goal of this project is to provide the community with a package that eases the learning curve for the advanced spectral timing techniques with a correct statistical framework.
 
-More details of current and planned capabilities are available in the `Stingray documentation <https://docs.stingray.science/intro.html#features>`_.
+More details of current and planned capabilities are available in the `Stingray documentation <https://docs.stingray.science/en/latest/#features>`_.
```

Comment:
Thanks @swapsha96! I think it's better to link to the stable version instead of the latest though. Latest is the development version on main, which is often not what people download through pip

Reply:
@matteobachetti I agree. I've updated the code.

Also, I noticed that `https://docs.stingray.science/` redirects to a version (I guess the latest version). I think it would be better if we redirect to the stable version. What do you say?

Reply:
It's a temporary measure, because the stable version, at the moment, displays a strange version number in the upper left. It will redirect to stable as soon  as we release 2.0